hubConf.py

```diff
@@ -0,0 +1,93 @@
+import cv2
+import torch
+import numpy as np
+import sys
+
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from baseline.utils.tensorboard import TensorBoard
+from baseline.Renderer.model import FCN
+from baseline.Renderer.stroke_gen import *
+
+from argparse import Namespace
+
+torch.manual_seed(1337)
+np.random.seed(1337)
+torch.backends.cudnn.deterministic = True
+torch.backends.cudnn.benchmark = False
+use_cuda = torch.cuda.is_available()
+
+class Model:
+    def __init__(self, device=None, jit=False):
+        self.device = device
+        self.jit = jit
+        self.criterion = nn.MSELoss()
+        net = FCN()
+        self.step = 0
+        self.opt = Namespace(**{
+            'batch_size': 64,
+            'debug': '',
+            'script': False,
+            })
+
+        if self.jit:
+            net = torch.jit.script(net)
+        self.module = net
+
+
+    def get_module(self):
+        return self.module
+
+    def train(self, niter=1):
+        self.module.train()
+        self.train_batch = []
+        self.ground_truth = []
+        for i in range(self.opt.batch_size):
+            f = np.random.uniform(0, 1, 10)
+            self.train_batch.append(f)
+            self.ground_truth.append(draw(f))
+
+        self.train_batch = torch.tensor(self.train_batch).float()
+        self.ground_truth = torch.tensor(self.ground_truth).float()
```

Comment:
need to move this to __init__

train_bach, .. = random
self.example_inputs = train_batch, ground_truth

Reply:
make sure to 
train_batch.to(device)
instead of if cuda .cuda()